packages/neon-core/src/rpc/parse.ts

```diff
@@ -50,16 +50,6 @@ export function StringParser(item: StackItemJson): string {
   return hexstring2str(item.value);
 }
 
-/**
- * Parses the result to a Fixed8.
- */
-export function Fixed8Parser(item: StackItemJson): number {

```

Comment:
if we are removing methods, then we should detail them in the changelog. The commit description is a little misleading as this isnt just docs anymore.